skyportal/handlers/api/followup_request.py

```diff
@@ -344,6 +344,19 @@ def get(self, followup_request_id=None):
               type: string
             description: |
               String to match status of request against
+          - in: query
+            name: numPerPage
+            nullable: true
+            schema:
+              type: integer
+            description: |
+              Number of followup requests to return per paginated request. Defaults to 10
```

Comment:
So say 50?

Reply:
I would say 100 just because I seem to recall that was the limit for candidates (and it cannot be too heavy, these are just followup requests). 
You may want a smaller default number for the front end elements.